src/main/java/de/tum/in/www1/artemis/domain/exam/StudentExam.java

```diff
@@ -25,6 +25,9 @@ public class StudentExam extends AbstractAuditingEntity {
     @Column(name = "submitted")
     private Boolean submitted;
 
+    @Column(name = "abandoned")
+    private Boolean abandoned;
```

Comment:
```suggestion
    private boolean abandoned = false;
```
Since the database uses a default, this can be used here as well to avoid having to deal with `Booleans.TRUE.equals(abondoned)` and similar.